### PR TITLE
Verification Successful Component

### PR DESCRIPTION
**Pull request by Ibhassan01**
Created a component for the "verification successful" overlay

### CODE REVIEW

File: src/components/VerificationSuccessful_comp.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function VerificationSuccessful(){
+    return (
+        <div className="space-y-6">
+            <div className="">
+                <div className="flex justify-between ">
+                    <h1 className="text-blue-500 font-bold text-5xl">Verification Successful</h1>
+                    <h2>cancel</h2>
```
Comment (annonymousauthority) [inline src/components/VerificationSuccessful_comp.js:7]: This isn't right. There's no `cancel` there and it's a button not a header

File: src/components/VerificationSuccessful_comp.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                </div>
+                <p className="my-6 text-2xl font-semibold">Account has been successfully verified with Carai</p>
+                <button className="px-6 py-3 bg-blue-600 my-7 text-white rounded-md font-bold border-[1px] w-full">
```
Comment (annonymousauthority) [inline src/components/VerificationSuccessful_comp.js:10]: Utilize `next/link` for this not a button. and it should point to the `/login` page

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/components/VerificationSuccessful_comp.js (hunk 1 above)
Comment (Ibhassan01) [reply src/components/VerificationSuccessful_comp.js:7]: The icon isn't ready yet so I used `cancel` instead

Comment (annonymousauthority) [reply src/components/VerificationSuccessful_comp.js:7]: Please change it from a header to a button, and you can leave the cancel text in there

File: src/components/VerificationSuccessful_comp.js (hunk 2 above)
Comment (annonymousauthority) [reply src/components/VerificationSuccessful_comp.js:10]: Please change this as well

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Comment by Ibhassan01** [comment]
Screenshot (19)

File: src/components/VerificationSuccessful_comp.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+function VerificationSuccessful(){
+    return (
+        <div className="space-y-6">
+            <div className="">
+                <div className="flex justify-between ">
+                    <h1 className="text-blue-500 font-bold text-5xl">Verification Successful</h1>
+                    <button>cancel</button>
+                </div>
+                <p className="my-6 text-2xl font-semibold">Account has been successfully verified with Carai</p>
+                <Link href="/login" className="hover:bg-blue-500 bg-violet-800 text-white text-center font-bold block w-full p-2 rounded-sm">
```
Comment (annonymousauthority) [inline src/components/VerificationSuccessful_comp.js:12]: change this to `rounded-lg` not `rounded-sm`

**Review by annonymousauthority — APPROVED** [review]
I'll merge when that change has been made

**Comment by Ibhassan01** [comment]
Screenshot (20)

**Comment by Ibhassan01** [comment]
Done

**Review by annonymousauthority — APPROVED** [review]
Please note this component will need cleaning when it comes to the colors

**Pull request merged**